package.json

```diff
@@ -6,22 +6,21 @@
   "license": "BSD-2-Clause",
   "version": "3.14.1",
   "engines": {
-    "node": ">=4.0.0"
+    "node": ">=6.0.0"
```

Comment:
Technically this is a breaking change too, but I guess nobody will complain here...